src/prefect/tasks/spacy/spacy_tasks.py

```
@@ -0,0 +1,204 @@
+            Task constructor
+    """
+
+    def __init__(
+        self,
+        text: str = u"",
+        nlp=None,
+        spacy_model_name: str = "en_core_web_sm",
+        disable: list = [],
+        component_cfg: dict = {},
+        **kwargs
+    ):
+        self.text = text
+
+        ## load spacy model
+        if nlp:
+            self.nlp = nlp
+        else:
+            try:
+                self.nlp = spacy.load(
+                    spacy_model_name, disable=disable, component_cfg=component_cfg
+                )
+            except IOError:
+                raise ValueError("spaCy model %s not found." % spacy_model_name)
+
+        super().__init__(**kwargs)
+
+    @defaults_from_attrs("text")
+    def run(self, text: str = u""):
+        """
+        Task run method. Creates a spaCy document.
+
+        Args:
+            - text (unicode, optional): text to be processed
+
+        Returns:
+            - Doc: spaCy document
+        """
+        doc = self.nlp(text)
+
+        return doc
+
+
+class SpacyTagger(Task):
+    """
+    Task for returning tagger from a spaCy pipeline.
+
+    Args:
+        - nlp (spaCy text processing pipeline, optional): a custom spaCy text
+            processing pipeline
+        - **kwargs (dict, optional): additional keyword arguments to pass to the
+            Task constructor
+    """
+
+    def __init__(self, nlp=None, **kwargs):
+        self.nlp = nlp
+        super().__init__(**kwargs)
+
+    @defaults_from_attrs("nlp")
+    def run(self, nlp=None):
+        """
+        Task run method. Returns tagger component of spaCy pipeline.
+
+        Args:
+            - nlp (spaCy text processing pipeline, optional): a custom spaCy text
+                processing pipeline, must be provided if not
+                specified in construction
+
+        Returns:
+            - Tagger: spaCy Tagger object
+
+        """
+        return nlp.tagger
```

I think that tasks should raise a value error if `nlp` is `None` (never passed in during run or initialization)

---

Added logic and tests in latest commit